tests/test_autoscaling/test_autoscaling.py

```diff
@@ -75,6 +75,17 @@ def test_create_autoscaling_group_within_elb():
         ),
     )
 
+    as_client.put_scheduled_update_group_action(
```

Comment:
Shouldn't there be some assertions related to this?

Reply:
Describe is implemented in a different branch. There is not much to assert on here. Done in this branch:  https://github.com/spulec/moto/compare/master...EBSCOIS:moto:feature/add-describe-scheduled-actions

Reply:
https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/autoscaling.html#AutoScaling.Client.describe_auto_scaling_groups